tests/NLog.UnitTests/LogFactoryTests.cs

```diff
@@ -182,7 +191,7 @@ public void SecondaryLogFactoryDoesNotTakePrimaryLogFactoryLock()
             {
                 bool threadTerminated;
 
-                var primaryLogFactory = LogManager.factory;
+                var primaryLogFactory = LogManager.factory;//QUESTION should this be new LogFactory()._syncRoot test instead???
```

Comment:
@snakefoot do you know the answer to this question?

Reply:
I think it is testing that the LogFactory owned by the LogManager can live together with a secondary LogFactory-instance. And there are no calls from the secondary LogFactory into the LogManager (and the primary LogFactory-instance)